drivers/adc/adc_shell.c

```diff
@@ -337,6 +337,12 @@ static int cmd_adc_print(const struct shell *sh, size_t argc, char **argv)
 			   adc->channel_config.acquisition_time,
 			   adc->channel_config.channel_id,
 			   adc->resolution);
+#if CONFIG_ADC_CONFIGURABLE_INPUTS
+	shell_print(sh, "Input positive: %u\n"
+			"Input negative: %u",
+			adc->channel_config.input_positive,
+			adc->channel_config.input_negative);
```

Comment:
Printing of the negative input only makes sense for differential channels.

Reply:
@anangl thanks, fixed, I've added the differential command.